packages/card/src/types.ts

```diff
@@ -0,0 +1,22 @@
+export type FzCardProps = {
+  title: string;
+  color?: FzCardColor;
```

Comment:
I'd advise against typing props as enum, as the developer is then forced to import the enum. Better off using a union type.

Reply:
I implemented the enum so that the end user is forced to use only the value defined within the enum. We should probably discuss this internally and choose what is best for us. For now, I will switch to the union type

Reply:
The union type lets you achieve the same without the need to import anything.
This was actually already discussed very early in the design system: https://github.com/fiscozen/design_system/pull/3#discussion_r1515771633